src/test/java/org/gridsuite/modification/server/BuildTest.java

```diff
@@ -482,7 +493,7 @@ private void testNetworkModificationsCount(UUID groupUuid, int actualSize) {
     public void tearDown() {
         Message<byte[]> message = null;
         try {
-            message = output.receive(TIMEOUT);
+            message = output.receive(4000);
```

Comment:
Why quitting from the names constant and change to *more* milliseconds in the teardown ?

Reply:
unit test fail with that timeout